noxfile.py

```diff
@@ -89,7 +92,15 @@ def lint(session: Any) -> None:
     session.run("isort", "--check", *SOURCE_FILES)
     session.run("black", "--check", *SOURCE_FILES)
     session.run("flake8", *SOURCE_FILES)
-    session.run("pylint", *SOURCE_FILES)
+    if (
```

Comment:
How is this to be used? Why do we need it behind a feature flag? I think we need a single `lint` task, and we can decide that we always want to do the `lint_per_folder` version.

Reply:
I thought a feature flag would be good 1) so I could submit this PR with it working the way I want without breaking the build, 2) to explicitly work on enabling the lints locally, test, fix, and then move. I can undo the FF and make the lint_per_folder work the same way as the setup.cfg version.

Reply:
i decided not to be such a wimp. removed the FF. I'll update setup.cfg as well